wp-content/themes/core/components/image/js/image.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @module
+ * @description Javascript for the image component.
+ */
+
+import lazySizes from 'lazysizes';
+
+/**
+ * Load unloaded lazy images to make sure they're present for printing.
+ */
+const loadBeforePrint = () => {
+	const lazyImages = document.querySelectorAll( 'img.lazyload' );
+	lazyImages.forEach( image => lazySizes.loader.unveil( image ) );
+};
+
+/**
+ * @function bindEvents
+ * @description Bind the events for this module here.
+ */
+const bindEvents = () => {
+	window.addEventListener( 'beforeprint', loadBeforePrint );
+};
+
+/**
+ * @function init
+ */
+const init = () => {
+	bindEvents();
+
+	console.info( 'SquareOne Theme: Initialized Image component.' );
+};
```

Comment:
Should we check to make sure there's lazy load images on the page before kicking off the script?

Reply:
Good question. I considered that and opted against it for 2 reasons: